src/helpers.ts

```diff
@@ -31,6 +31,11 @@ export function getIbdBlockHeight(): number | undefined {
   }
 }
 
+export function isBlockInIbdMode(blockHeight: number): boolean {
+  const ibdBlockHeight = getIbdBlockHeight();
```

Comment:
I think you should include the `isEventReplay` property from `PgStore` here because event replay does not necessarily mean IBD if I remember correctly

Reply:
Ahh that's right, fixed!